src/components/verify/verify-page.tsx

```diff
@@ -77,9 +78,10 @@ export const VerifyPage: React.FunctionComponent = () => {
           network: NETWORK_NAME,
           promisesCallback: (promises) => {
             const HASH_STATUS_INDEX = 0;
-            const TOKEN_REGISTRY_STATUS_INDEX = 2;
-            const DOCUMENT_STORE_STATUS_INDEX = 3;
-            const DNS_TXT_VERIFIER_INDEX = 4;
+            const TOKEN_REGISTRY_STATUS_INDEX = 1;
+            const DOCUMENT_STORE_STATUS_INDEX = 2;
+            const DNS_TXT_VERIFIER_INDEX = 3;
+            const ALLOWED_ISSUERS_VERIFIER_INDEX = 6;
```

Comment:
wow this is really hard to read... It seemed like one has to know the order of the default verifiers in the oa-verify package and if the order is swapped there, it will break the code here......

Reply:
yeah it is and there is not better way as for now to check for individual verifier status, keeping like this (also out of scope)